recipes/Drop-seq_tools/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "Drop-seq_tools" %}
```

Comment:
Can we have small letters here?

Reply:
the name is used to build the file name to download from their web site. I'd rather keep it like this if possible.